index.js

```diff
@@ -1,82 +1,86 @@
-const path = require('path');
-const { Transformer } = require('@parcel/plugin');
-const { default: SourceMap } = require('@parcel/source-map');
-const { relativeUrl } = require('@parcel/utils');
-const { compile, preprocess } = require('svelte/compiler.js');
+const path = require("path");
+const { Transformer } = require("@parcel/plugin");
+const { default: SourceMap } = require("@parcel/source-map");
+const { relativeUrl } = require("@parcel/utils");
+const { compile, preprocess } = require("svelte/compiler.js");
+const { exit } = require("process");
```

Comment:
This can be removed. I was using process.exit to debug under certain conditions where logging output was not being shown.